client/src/cordova/android/OutlineAndroidLib/outline/src/main/java/org/outline/vpn/VpnTunnelService.java

```diff
@@ -263,15 +215,9 @@ private synchronized ErrorCode startTunnel(
       }
     }
 
-    final shadowsocks.Config configCopy = new shadowsocks.Config();
-    configCopy.setHost(config.proxy.host);
-    configCopy.setPort(config.proxy.port);
-    configCopy.setCipherName(config.proxy.method);
-    configCopy.setPassword(config.proxy.password);
-    configCopy.setPrefix(config.proxy.prefix);
     final shadowsocks.Client client;
     try {
-      client = new shadowsocks.Client(configCopy);
+      client = new shadowsocks.Client(config.transportConfig);
```

Comment:
This is great! 

I think we can also rename client/go/outline/shadowsocks into a package that is not shadowsocks specific. The Methods like func NewClientFromJSON(configJSON string) (*Client, error) are already non SS specific.